src/ocl/convolutionocl.cpp

```diff
@@ -1012,16 +863,11 @@ void GetSolutions(Handle& handle,
         }
 
         // To stop crashes with old ufdb file
-        if(solver_id == solver::Id::gemm() && !problem.direction.IsBackwardWrW())
+        if(solver_id == solver::Id::gemm())
```

Comment:
we can deal with this more efficiently at the parse time and not pollute the solver namespace.

Reply:
> Vasilii Filippov for the rest of the GEMM solvers, we adopted `GEMMRest` naming convention. Can we adhere to the same and not use a new name (`GemmWrwUniversal`) ?

Basically, I regret the decision to name the rest, because as far as I understand they are not, and not using them when anything is available is a kind of performance heuristics.

But I can rename this to rest as well if you wish.

Reply:
I'd rather stay with a simple solution and remove this completely, when we have system data base ready to remove it.